windows/benji.py

```diff
@@ -99,13 +99,14 @@ def events(frame,put):
 				print("By ", status.user.screen_name, " at ", status.user.created_at)
 				
 	#Get friends from twitter
-	elif link[-3] == "follow" and link[-1] == "twitter":
-		auth = OAuthHandler(twitterCredentials.consumer_key, twitterCredentials.consumer_secret)
-		auth.set_access_token(twitterCredentials.access_token, twitterCredentials.access_secret)
-		api = tweepy.API(auth)
-		for friend in tweepy.Cursor(api.friends).items():
-			print("\nName: ", json.dumps(friend.name), " Username: ", json.dumps(friend.screen_name))
-	
+	elif len(link[-1]) == "twitter":

```

Comment:
This is logically not correct `len(link[-1]) will give you an integer, which cannot be equal to a string value.`
What was the intended change? I would prefer it be reverted to the old if condition.

Reply:
Sorry, it was a mistake.